.oxlintrc.json

```diff
@@ -0,0 +1,329 @@
+{
+  "$schema": "./node_modules/oxlint/configuration_schema.json",
+  "plugins": [
+    "import",
+    "jsdoc",
+    "unicorn"
+  ],
+  "categories": {
+    "correctness": "off"
+  },
+  "env": {
+    "es2021": true,
+    "browser": true,
+    "node": true
+  },
+  "ignorePatterns": [
+    "packages/schema/schema/**",
+    "packages/nuxt/src/app/components/welcome.vue",
+    "packages/nuxt/src/app/components/error-*.vue",
+    "packages/nuxt/src/core/runtime/nitro/templates/error-*",
+    "**/node_modules",
+    "**/jspm_packages",
+    "**/package-lock.json",
+    "packages/*/README.md",
+    "!packages/nuxi/README.md",
+    "packages/*/LICENSE",
+    "docs/LICENSE",
+    "*/**/yarn.lock",
+    ".yarn",
+    "**/*.log",
+    "**/.temp",
+    "**/.tmp",
+    "**/.cache",
+    "**/.lycheecache",
+    "**/dist",
+    "**/.nuxt",
+    "**/.nuxt-*",
+    "**/.vercel",
+    "**/.netlify",
+    "**/.output",
+    "**/.output-*",
+    "**/.gen",
+    "**/reports",
+    "**/coverage",
+    "**/*.lcov",
+    "**/.nyc_output",
+    "**/.vscode",
+    "**/*.iml",
+    "**/.idea",
+    "!.idea/nuxt.iml",
+    "!.idea/modules.xml",
+    "!.idea/inspectionProfiles/Project_Default.xml",
+    "**/.DS_Store",
+    "**/.AppleDouble",
+    "**/.LSOverride",
+    "**/.DocumentRevisions-V100",
+    "**/.fseventsd",
+    "**/.Spotlight-V100",
+    "**/.TemporaryItems",
+    "**/.Trashes",
+    "**/.VolumeIcon.icns",
+    "**/.com.apple.timemachine.donotpresent",
+    "**/.AppleDB",
+    "**/.AppleDesktop",
+    "**/Network Trash Folder",
+    "**/Temporary Items",
+    "**/.apdisk",
+    "**/.vercel_build_output",
+    "**/.build-*",
+    "**/.env",
+    "**/fixtures-temp",
+    "**/.pnpm-store",
+    "**/eslint-typegen.d.ts",
+    "**/.eslintcache",
+    "**/test-results/",
+    "**/playwright-report",
+    "**/temp",
+    "**/public"
+  ],
+  "globals": {
+    "$fetch": "readonly",
+    "NodeJS": "readonly",
+    "__dirname": "readonly",
+    "__filename": "readonly",
+    "Buffer": "readonly",
+    "clearImmediate": "readonly",
+    "process": "readonly",
+    "setImmediate": "readonly",
+    "computed": "readonly",
+    "defineEmits": "readonly",
+    "defineExpose": "readonly",
+    "defineProps": "readonly",
+    "onMounted": "readonly",
+    "onUnmounted": "readonly",
+    "reactive": "readonly",
+    "ref": "readonly",
+    "shallowReactive": "readonly",
+    "shallowRef": "readonly",
+    "toRef": "readonly",
+    "toRefs": "readonly",
+    "watch": "readonly",
+    "watchEffect": "readonly"
+  },
```

Comment:
is there any way to get oxlint to be aware of this from the TS environment?

Reply:
Will check